src/print.c

```diff
@@ -656,7 +656,13 @@ int pr_date(date_t *d, int mode){
 	d_fp = print_prep(DATE, 0);
 
     PR_STRT(d_fp);
+ #ifdef YMD_DATE
+    char ymd_date[D_DATE_LEN];
+    PR_DATE(ymd_date, d->year-1900, d->monthnuminyear, d->daynuminmonth);
+    PR_STR(d_fp, ymd_date, D_DATE_LEN);
+#else   
     PR_INT(d_fp, d->datekey);
```

Comment:
I'm going to just change this to the new logic as it will work for both default and YMD_DATE format since the PR_DATE macro already has the ifdef and so we can just remove here.